src/main/scala/no/ndla/imageapi/ImageSwagger.scala

```diff
@@ -19,15 +19,15 @@ class ResourcesApp(implicit val swagger: Swagger) extends ScalatraServlet with N
 
 object ImagesApiInfo {
   val apiInfo = ApiInfo(
-  "Images Api",
-  "Documentation for the Images API of NDLA.no",
+  "Image API",
+  "Services for accessing images",
   "https://ndla.no",
   ImageApiProperties.ContactEmail,
   "GPL v3.0",
   "http://www.gnu.org/licenses/gpl-3.0.en.html")
 }
 
-class ImageSwagger extends Swagger("2.0", "0.8", ImagesApiInfo.apiInfo) {
+class ImageSwagger extends Swagger("2.0", "1.0", ImagesApiInfo.apiInfo) {
```

Comment:
Dette sier hvilke versjoner av swagger som er støttet?

Reply:
Den første ja. Den jeg har endret er "vår" versjonering av api'et. Følte det var tid for å gå til 1.0 nå... :)